app/backend/src/couchers/models.py

```diff
@@ -239,6 +241,8 @@ class User(Base):
     # e.g. cus_JjoXHttuZopv0t
     stripe_customer_id = Column(String, nullable=True)
 
+    new_notifications_enabled = Column(Boolean, nullable=False, server_default=text("false"))
```

Comment:
Why is this `text("false")`?

Reply:
Postgres will take `false` as a string. I don't think `False` works, and this is the same way we do it elsewhere.

Reply:
Just above, it uses a plain string without the Text function. But doesn't matter